http/http-minimum-reactive/src/test/java/io/quarkus/ts/http/minimum/reactive/OperatorResourceIT.java

```diff
@@ -0,0 +1,26 @@
+
+package io.quarkus.ts.http.minimum.reactive;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.http.ContentType;
+
+@QuarkusIntegrationTest
+public class OperatorResourceIT {
+
+    @Test
+    void executePostOperator() {
```

Comment:
thought about it a bit and I think you would achieve same effect if you added this method to any other test class?